07/hangman/hangman_score.py

```diff
@@ -0,0 +1,17 @@
+from hangman_img import obesenec
+
+
+def score(retezec, pismeno, pokus, slovo):
+
+    if pismeno in slovo:
```

Comment:
Ak užívateľ trafí posledné písmeno tak mu vypíšeš reťazec ale nedozvie sa ako hra dopadla. Napr. takto:
```
kobra
None
```
Po tom ako doplní všetky písmená alebo vyčerpá všetky pokusy by malo dôjsť k vyhodnoteniu hry. Skôr to nemá význam, stačí len po každom pokuse printovať aktuálny reťazec a zároveň ideálne aj aktuálneho obesenca.